packages/typescriptlang-org/src/templates/scripts/setupLikeDislikeButtons.ts

```diff
@@ -4,14 +4,6 @@ export const setupLikeDislikeButtons = (slug: string, i: any) => {
   if (!likeButton || !dislikeButton) return
 
   const clicked = (eventName: string) => () => {
-    // @ts-ignore
```

Comment:
Honestly, I'm wondering if we should be ditching the buttons altogether for now, at least until we can move to something different (or the same but recreated). Otherwise we'll have thumbs up/down which just don't do anything. But maybe that's a little too invasive for the short term. But we definitely can't leave them.

Reply:
Thankfully removing it doesn't seem too hard; happy to push that up unless you want to do it.

Reply:
Or, actually, we should just make them not show up; that's a lot easier to undo.

Reply:
(so, ctrl-f `dislike` and comment out, more or less)

Reply:
I've commented out the buttons. `setupLikeDislikeButtons.ts` creates these, so I can comment that out too if needed.

Reply:
Looks good; that's what I had tried locally too.